examples/typescript/aws-cloudfront-proxy/main.ts

```diff
@@ -46,7 +45,7 @@ class MyStack extends TerraformStack {
       allowOverwrite: true,
     });
 
-    new AcmCertificateValidation(this, "certvalidation", {
+    new ACM.AcmCertificateValidation(this, "certvalidation", {
```

Comment:
What do you think about the duplicated prefix?

Reply:
I'm a bit torn. As the example is currently setup I'd prefer to remove the prefix, but thinking about Java/C# where the import/using is more specific would lead to just seeing `new CertificateValidation`. That's not so bad, but other resources get awfully generic names once the prefix is stripped away. Also increases the likelihood of having a name that conflicts with something else.

Reply:
I'll leave the complete part for now. There are some groups that have a lot of resources not prefixed (e.g. VPC) where removing the prefix would not do anything / remove the resource entirely. This leads to pretty unpredictable behaviour. If we want to go that route we should plan it more carefully I believe